casepro/profiles/context_processors.py

```diff
@@ -8,11 +8,14 @@ def user(request):
     if request.user.is_anonymous() or not request.org:
         is_admin = False
         partner = None
+        is_faq_only = True
     else:
         is_admin = request.user.can_administer(request.org)
         partner = request.user.get_partner(request.org)
+        is_faq_only = request.user.must_use_faq()
 
     return {
         'user_is_admin': is_admin,
-        'user_partner': partner
+        'user_partner': partner,
+        'user_is_faq_only': is_faq_only
```

Comment:
I get now why this was originally a context processor because otherwise there's no easy way to pass it to the reply modal template